src/cluster.c

```diff
@@ -219,6 +223,32 @@ int auxShardIdPresent(clusterNode *n) {
     return strlen(n->shard_id);
 }
 
+int auxHumanNodenameSetter(clusterNode *n, void *value, int length) {
+    if (n && !strcmp(value, n->human_nodename)) {
```

Comment:
```suggestion
    if (n && !strncmp(value, n->human_nodename, length)) {
```
We know these are null terminated strings, but people like to report that we aren't using "safe" functions.

Reply:
I have replaced strcmp with strncmp as suggested.